src/data/actions.ts

```diff
@@ -3,16 +3,16 @@ import { primaryLabelForIssue } from './labels';
 
 // Based on state, what do we need to tell the API to update
 const updateForAction = (action: any, labelToRemove: any) => {
-  
+
   // Compose our label names
   const labelNameToAdd = `${action.name} ${action.icon}`
   const labelNameToRemove = `${labelToRemove.name} ${labelToRemove.icon}`;
 
   // Handle different actions differently
   if ( action.icon === '✅' ) {
-    const removeWhenClosing = ['Doing ⏳', 'Review 👓', 'Blocked 🛑', 'Paused ⏸', ...[labelNameToRemove]];
```

Comment:
this was causing priority labels and OOO labels to be removed when an issue was marked as done without going through doing first